src/app/ReadHandler.h

```diff
@@ -354,17 +351,14 @@ class ReadHandler : public Messaging::ExchangeDelegate
 
     bool IsIdle() const { return mState == HandlerState::Idle; }
 
-    // TODO (#27672): Change back to IsReportable once ReportScheduler is implemented so this can assess reportability without
-    // considering timing. The ReporScheduler will handle timing.
-    /// @brief Returns whether the ReadHandler is in a state where it can immediately send a report. This function
+    /// @brief Returns whether the ReadHandler is in a state where it can send a report. This function
     /// is used to determine whether a report generation should be scheduled for the handler.
-    bool IsReportableNow() const
+    bool IsReportable() const
     {
-        // Important: Anything that changes the state IsReportableNow depends on in
-        // a way that causes IsReportableNow to become true must call ScheduleRun
+        // Important: Anything that changes the state IsReportable depends on in
+        // a way that causes IsReportable to become true must call ScheduleRun
```

Comment:
It's not supposed to call ScheduleRun.  It's supposed to call something on the callback, no?

Reply:
Correct